packages/ui/src/components/Modal/style.less

```diff
@@ -87,9 +90,54 @@
       padding: @condo-modal-half-padding-without-border @condo-modal-padding @condo-modal-half-padding;
       border-top: @condo-global-border-width-default solid @condo-modal-header-scrolling-border-bottom-color;
 
+      display: flex;
+      flex-wrap: wrap;
+      row-gap: @condo-global-spacing-16;
+      column-gap: @condo-global-spacing-16;
+      justify-content: end;
+
       .condo-btn + .condo-btn:not(.condo-dropdown-trigger, :first-child) {
         margin-left: 0;
       }
     }
+
+    @media screen and (max-width: @condo-global-breakpoints-tablet-large) {
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      top: initial;
+      max-width: 100%;
+      width: auto !important;
+      margin: @condo-global-spacing-16;
+
+      .condo-modal-header {
+        padding:
+                @condo-modal-mobile-spacing
+                @condo-modal-mobile-header-padding-right
+                @condo-modal-mobile-spacing
+                @condo-modal-mobile-spacing;
+      }
+
+      .condo-modal-close {
+        top: @condo-modal-mobile-spacing;
+        right: @condo-modal-mobile-spacing;
+      }
+
+      .condo-modal-body {
+        margin: 0 @condo-modal-mobile-spacing;
+        padding: 0 0 @condo-modal-mobile-spacing;
+      }
+
+      .condo-modal-footer {
+        padding: @condo-modal-mobile-spacing;
+
+        flex-direction: column;
+
+        button {
```

Comment:
use `.condo-` notation, you should not control elements from other design systems...

Reply:
and button with a href is `<a/>` btw